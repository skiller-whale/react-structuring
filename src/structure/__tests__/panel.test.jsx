import React from 'react'
import {Panel} from '../App'
import renderer from 'react-test-renderer'

test('Panel renders heading and children correctly', () => {
  const component = renderer.create(
    <Panel heading="A tweet">
      <span>Some content</span>
      <img src="some_image.jpg"/>
    </Panel>
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
