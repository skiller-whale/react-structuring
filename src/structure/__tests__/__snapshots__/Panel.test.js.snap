// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel renders heading and children correctly 1`] = `
<div
  className="panel panel-default"
>
  <div
    className="panel-heading"
  >
    <h3
      className="panel-title"
    />
  </div>
  <span>
    Some content
  </span>
  <img
    src="some_image.jpg"
  />
</div>
`;
